nh_eobs_mobile/static/src/js/nhlib.js

```diff
@@ -1226,7 +1230,12 @@ NHMobilePatient = (function(superClass) {
         chartFunc(self, serverData);
       }
       if (validTable) {
-        return tableFunc(self, serverData);
+        tableFunc(self, serverData);
+      }
+      if (activeTab === "#graph-content") {
```

Comment:
How come you use `===` here and `is` earlier? I believe they are the same for string comparisons, but it's good to be consistent.